create-neon/data/templates/README.md.hbs

```diff
@@ -1,5 +1,82 @@
 # {{package.name}}
 
 {{#if package.description}}
-{{package.description}}
+*{{package.name}}:* {{package.description}}
+
 {{/if}}
+This project was bootstrapped by [create-neon](https://www.npmjs.com/package/create-neon).
+
+## Building {{package.name}}
+
+Building {{package.name}} requires a [supported version of Node and Rust](https://github.com/neon-bindings/neon#platform-support).
+
+You can build the project with npm. In the project directory, run:
+
+```sh
+$ npm install
+```
+
+## Exploring {{package.name}}
+
+After building {{package.name}}, you can explore its exports at the Node REPL:
+
+```sh
+$ npm install
+$ node
+> require('.').hello()
+"hello node"
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm install`
+
+Builds the project from source. 
+
+### `npm test`
+
+Runs the unit tests by calling `cargo test`. You can learn more about [adding tests to your Rust code](https://doc.rust-lang.org/book/ch11-01-writing-tests.html) from the [Rust book](https://doc.rust-lang.org/book/).
+
+## Project Layout
+
+The directory structure of this project is:
+
+```
+{{package.name}}/
+├── Cargo.toml
+├── README.md
+├── index.node
+├── package.json
+└── src
+    └── lib.rs
+```
+
+### Cargo.toml
+
+The Cargo [manifest file](https://doc.rust-lang.org/cargo/reference/manifest.html), which informs the `cargo` command.
+
+### README.md
+
+This file.
+
+### index.node
+
+The binary module generated by building the project.
```

Comment:
Can you add a bit explaining that this is a C dynamic shared library generated by `cargo` and that it's a copy of the file in `target/`?